packages/arb-token-bridge-ui/src/util/wagmi/wagmiAdditionalNetworks.ts

```diff
@@ -1,5 +1,22 @@
 import { Chain, sepolia as sepoliaDefault } from 'wagmi'
-import { ChainId, explorerUrls, rpcURLs } from '../networks'
+import { ChainId, ChainWithRpcUrl, explorerUrls, rpcURLs } from '../networks'
+
+export function chainToWagmiChain(chain: ChainWithRpcUrl): Chain {
```

Comment:
NIT: Renaming this to `customChainToWagmiChain` or `orbitChainToWagmiChain` might make more sense?

Reply:
can do, although it can be used for any chain